TodoApi/OpenTelemetryExtensions.cs

```diff
@@ -12,10 +13,10 @@ public static WebApplicationBuilder AddOpenTelemetry(this WebApplicationBuilder
     {
         var resourceBuilder = ResourceBuilder.CreateDefault().AddService(builder.Environment.ApplicationName);
 
-        builder.Logging.AddOpenTelemetry(o =>
+        builder.Logging.AddOpenTelemetry(logging =>
         {
-            // TODO: Setup an exporter here
-            o.SetResourceBuilder(resourceBuilder);
+            logging.SetResourceBuilder(resourceBuilder)
+                   .AddOtlpExporter();
```

Comment:
This doesn't require anything to be running?

Reply:
Wait why is that?

Reply:
I believe so, yes. I'm looking through the source code at the moment to verify.

Here's the Exporter in question:
https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpLogExporter.cs#L31

Reply:
Where is it sending these logs if nothing is configured?

Reply:
With the current configuration it'll send the logs to `localhost:4317` using gRPC. It fails gracefully when nothing is there to receive them.

Should we perhaps leave this bit out, until vendor support improves?

Reply:
Leave it commented out with a comment.

Reply:
Done 👍 
https://github.com/davidfowl/TodoApi/pull/11/commits/e5308f7c611689e40ad69256d7b59c488d8a4b54